packages/components/src/components/hds/layout/grid/index.ts

```diff
@@ -24,11 +24,20 @@ export const ALIGNS: HdsLayoutGridAligns[] = Object.values(
 export const DEFAULT_GAP = HdsLayoutGridGapValues.Zero;
 export const GAPS: HdsLayoutGridGaps[] = Object.values(HdsLayoutGridGapValues);
 
+type ResponsiveColumnWidths = {
+  sm?: string;
+  md?: string;
+  lg?: string;
```

Comment:
[question] I don't have a problem with these breakpoints necessarily, but is it worth including all the way up to xl and xxl to match the rest of the [breakpoint docs](https://helios.hashicorp.design/foundations/breakpoints)?

Reply:
I added in "xl" & "xxl" breakpoints now. I may refactor how they work exactly, but for now I have them working as the other originally implemented ones.

Reply:
There's an issue with the Large view now that I'll work on...

Reply:
Fixed some typos in variable names so everything should be working again now